tests/e2e/pageobjects/dashboard/CreateWorkspace.ts

```diff
@@ -46,7 +48,18 @@ export class CreateWorkspace {
 
         const sampleLocator: By = this.getSampleLocator(sampleName);
 
-        await this.driverHelper.waitAndClick(sampleLocator, timeout);
+        try {
+            await this.driverHelper.waitAndClick(sampleLocator, timeout);
```

Comment:
The "StaleElementReferenceError" means that driver has a link to the object and tries to interact with this object but link is expired because page is dynamic and old object doesn't exist more. You don't need to refresh page, you just need to get this object again (obtain valid link ), and this link already implemented inside the "waitAndClick" method.
```
            try {
                await element.click();
                return;
            } catch (err) {
                if (err instanceof error.StaleElementReferenceError) {
                    Logger.debug(`DriverHelper.waitAndClik - ${elementLocator} - StaleElementReferenceError - ${err}`);
                    await this.wait(polling);
                    continue;
                }

                Logger.error(`DriverHelper.waitAndClick - failed with an unexpected exception - ${err}`);
                throw err;
            }
            ```

Reply:
If problem really with "Stale elementReferenceError" I would suggest you try to play with timeouts or take a look to the issue once more. Because right now it looks like duplication.

Reply:
the issue here is that the dashboard was constantly stuck in the "loading" state with half of the metadata missing and the dom was constantly changing.
refreshing the page fixed the issue for us, so I would prefer to keep this change here. It's a stale version of che anyway, won't influence main or later releases

Reply:
Yup, make sense, thank you.